job-service/pom.xml

```diff
@@ -16,237 +16,264 @@
 
```

Comment:
Why is there so much change in this file, it is showing huge changes. Please clean this up so it makes it possible to review.

Reply:
Fixed